tests/virtualization/universal/register_guests.pm

```diff
@@ -16,13 +16,20 @@ use version_utils;
 sub run_test {
     my ($self) = @_;
     # Use serial terminal, unless defined otherwise. The unless will go away once we are certain this is stable
-    $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    # $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    select_console 'root-console';
+    my %ltss_products = @{get_var_array("LTSS_PRODUCTS")};
 
     foreach my $guest (keys %virt_autotest::common::guests) {
         record_info "$guest", "Registrating $guest against SMT";
         my ($sles_running_version, $sles_running_sp) = get_os_release("ssh root\@$guest");
+        my $version_id = $sles_running_sp != 0 ? "$sles_running_version.$sles_running_sp" : $sles_running_version;
         if ($sles_running_version >= 12) {
             assert_script_run("ssh root\@$guest SUSEConnect -r " . get_var('SCC_REGCODE') . " -e " . get_var("SCC_EMAIL"));
+            sleep 10;
+            assert_script_run("ssh root\@$guest SUSEConnect -p SLES-LTSS/$version_id/x86_64 -r " . $ltss_products{"$version_id"}) if ($ltss_products{"$version_id"});
+            sleep 10;
+            record_info($ltss_products{"$version_id"});
```

Comment:
Do we need the scc codes of LTSS products is shown so noticeable? Of course, this may be not a problem just preference.

Reply:
@tonyyuan1  Agree, we'd better not explicitly show it in record_info. And even better, you can rename LTSS_PRODUCTS to LTSS_PRODUCTS_REGCODE_SECRET, so that in jobs' vars.json and settings webpage, it will not be displayed.

Reply:
They are key test data in this test.  It's helpful to find out some hiding errors which don't even fail/soffail test. 
It might also help reviewers/QA to understand this test quickly to save their time. 
Yes, it's not good to scc code. I will remove them.

Isn't LTSS_PRODUCTS_REGCODE_SECRET too lengthy?

Reply:
Yes, a little bit, but SECRET is necessary to hide it, how about LTSS_REGCODE_SECRET? Up to you to decide better ones.

Reply:
LTSS_REGCODES_SECRET is fine